data/utils/zerion.ts

```diff
@@ -63,4 +63,20 @@ export async function getPortfolio(address: string): Promise<number> {
   })
 
   return portfolio.payload.portfolio.total_value
+}
+
+export async function getDGTreasury() {
```

Comment:
Hey, this function doesn't belong in zerion.ts, can you move it to decentralgames.ts?